server/api/views.py

```diff
@@ -31,3 +37,38 @@ def create_user(user_data: dict, session=Depends(get_session)):
     session.commit()
     session.refresh(user)
     return user
+
+
+@app.post("/api/workflows/{workflow_id}/run", status_code=200)
```

Comment:
Add a `tags=["workflows"]` argument to the decorator. This is useful so the endpoint can be grouped in the documentation UI (when you go to `localhost:8000/docs`) and is also useful when I finish setting up the autogenerated typescript client for this API. (Look at the `/users` endpoint in this file for an example of the `tags` argument)